packages/components/src/internal/components/domainproperties/samples/SampleTypeDesigner.tsx

```diff
@@ -566,6 +620,17 @@ class SampleTypeDesignerImpl extends React.PureComponent<Props & InjectedBaseDom
                     domainFormDisplayOptions={domainFormDisplayOptions}
                 />
                 {error && <div className="domain-form-panel">{error && <Alert bsStyle="danger">{error}</Alert>}</div>}
+                {showUniqueIdConfirmation && (
+                    <ConfirmModal
+                        title={'Updating Sample Type with Unique ID field' + (numNewUniqueIdFields !== 1 ? 's' : '')}
```

Comment:
I assume it is very unlikely to change, but should we use the unique ID data type display value variable in this string for "Unique ID"?